### PR TITLE
feat: add bold and tone props for label in MaTextField

### PR DESCRIPTION
**Pull request by sergigraciaj**
As proposed in the design for the new solar product

image

### CODE REVIEW

**Review by alexcastelo95 — APPROVED** [review]
Looks Good Sergi 👍🏽

**Comment by afontcu** [comment]
Given this and the fact that @Samitier mentioned the need to render a tone="white" for a MaOption, what if we simply allowed all our Form elements to accept tone/weight props, just like MaText/Heading does, so we kill all these requirements at once? Needing to pass extra props to pretend we don't need to style the inner MaText of these components feels a bit unnatural

File: src/components/MaTextField/MaTextField.vue [hunk 1]
```diff
@@ -7,6 +7,8 @@
         size="small"
         tag="label"
         class="ma-text-field__label"
+        :tone="labelTone"
+        :bold="isLabelBold"
```
Comment (afontcu) [inline src/components/MaTextField/MaTextField.vue:11]: Looks like props got renamed but not these ones (smells like a missing test 😉)

Comment (sergigraciaj) [reply src/components/MaTextField/MaTextField.vue]: 🙈 
thx!

**Review by afontcu — APPROVED** [review]
LGTM – make sure you update the title accordingly!

**Pull request merged**